lms/static/sass/views/_login-register.scss

```diff
@@ -67,7 +67,7 @@ $sm-btn-linkedin: #0077b5;
         }
     }
 
-    form {
+    form, .other-login-wrapper {
         border: 1px solid $gray-l4;
         border-radius: 5px;
         padding: 0px 25px 20px 25px;
```

Comment:
Like that pop song the kids are listening to, we're all about the ``$baseline``. Mind syncing up these values to use that variable.